tests/smoke/CMakeLists.txt

```diff
@@ -118,7 +118,16 @@ if (ENABLE_CUDA)
     blt_add_test(NAME blt_cuda_runtime_smoke
                  COMMAND blt_cuda_runtime_smoke)
 
-
+    if (ENABLE_OPENMP)
```

Comment:
Would it make sense to test ``cuda_runtime`` + ``openmp`` ?

Reply:
Honestly kind of torn.. On one hand I don't want a million smoke tests but then it might also fail.